fleet/lib/prometheus-alertrules/rules/ceph.yaml

```diff
@@ -0,0 +1,169 @@
+groups:
+  - name: "ceph.rules"
```

Comment:
See also: https://github.com/lsst-it/k8s-cookbook/pull/346#discussion_r1632858849

My general goal/intention is to have the charts still deploy the alerts to the cluster, so they become visible in Prometheus rule interface (on http://prometheus/alerts) and Alertmanager (on http://alertmanager), but not route the alerts. A lot of helm generally only allow for "all or nothing" label modifications for alert routing.

Having them present in the web interfaces allows us to check which alerts we care about and are actually doing stuff, after which we can duplicate them into this fleet bundle to deploy and modify them to match our own setup.

The modification is sometimes needed because the helm chart rules are unaware of the deployed architecture of Prometheus/Mimir. In our case, the `prom_cluster` external label for example.

Reply:
I'm not following w.r.t. the `prom_cluster` label. Under the current architecture of prometheus evaluating rules on each k8s cluster, the `prom_cluster` label does not exist at the time of evaluation as its an external label applied as part of the remote write to mimir.  Is this in preparation to switch to using mimir ruler and evaluating rules centrally?

Reply:
https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/#templating suggests that `$externalLabels` is accessible during rule evaluation.  I've been unable to find an example of it being used.